src/index.js

```diff
@@ -1,12 +1,38 @@
-const express = require("express")
+/* eslint-disable no-console */
+// TODO: Do we still need this.
+import 'source-map-support/register'
+
+// Do we still use regenerator?
+import "regenerator-runtime/runtime"
```

Comment:
I'm about 90% sure this is safe to remove, and was only necessary for https://github.com/relay-tools/react-relay-network-modern#what-if-regeneratorruntime-is-not-defined, which i'm also about 90% sure has been fixed... We should try to ditch it and click around the new pages a bit.

Reply:
You should also check w/ different browsers just to be safe.

Reply:
@icirellik Can we delete this and see what happens?